lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -475,22 +476,43 @@ ImageryLayerCatalogItem.enableLayer = function(catalogItem, imageryProvider, opa
         isRequiredForRendering: catalogItem.isRequiredForRendering,
         onLoadError: function(e) {
             if (defined(layer) && (layer.show === true || !defined(layer.show))) {
-                globeOrMap.terria.error.raiseEvent(new TerriaError({
-                    sender: catalogItem,
-                    title: 'Error accessing catalogue item',
-                    message: '\
+
+                if (catalogItem === catalogItem.terria.baseMap ||
+                    catalogItem.terria.baseMap instanceof CompositeCatalogItem && catalogItem.terria.baseMap.items.indexOf(catalogItem) >= 0) {
+
+                    globeOrMap.terria.error.raiseEvent(new TerriaError({
+                        sender: catalogItem,
+                        title: 'Error accessing base map',
+                        message: '\
+An error occurred while attempting to download tiles for base map ' + catalogItem.terria.baseMap.name + '.  This may indicate that there is a \
+problem with your internet connection, that the base map is temporarily unavailable, or that the base map \
+is invalid.  Please select a different base map using the Map button in the top-right corner.  Further technical details may be found below.<br/><br/>\
+<pre>' + formatError(e) + '</pre>'
+                    }));
+
+                    layer.show = false;
+                    catalogItem.terria.baseMap = undefined;
+
+                    // Don't use this base map again on startup.
+                    catalogItem.terria.setLocalProperty('basemap', undefined);
+                } else {
+                    globeOrMap.terria.error.raiseEvent(new TerriaError({
+                        sender: catalogItem,
+                        title: 'Error accessing catalogue item',
+                        message: '\
```

Comment:
You can test this by switching to the Australian Hydrography base map, which is currently broken.  `=== 0` would assume that the failing layer is the bottom-most, which is not true when the base is a composite like Hydrography and the upper layer is the one that fails.